src/sentry/integrations/issue_alert_image_builder.py

```diff
@@ -18,9 +20,18 @@
 from sentry.snuba.referrer import Referrer
 from sentry.types.integrations import ExternalProviderEnum
 from sentry.utils import metrics
-from sentry.utils.performance_issues.detectors.utils import escape_transaction
+from sentry.utils.locking import UnableToAcquireLock
+from sentry.utils.locking.backends import LockBackend
+from sentry.utils.locking.manager import LockManager
+from sentry.utils.performance_issues.detectors import utils
+from sentry.utils.services import build_instance_from_options_of_type
 
 logger = logging.getLogger("sentry.chartcuterie")
+locks = LockManager(
+    build_instance_from_options_of_type(
+        LockBackend, settings.SENTRY_POST_PROCESS_LOCKS_BACKEND_OPTIONS
```

Comment:
Is this the right options to use?

Reply:
This lock util is not widely used in codebase like I'm the only person using the blockacquire which seems a bit scary.

Reply:
I'll let you figure it out, I'm not exactly sure.

Reply:
Updated to the correct one.